packages/design/src/colors.css

```diff
@@ -8,6 +8,8 @@
   --color-destructive--hover: var(--color-red--dark);
   --color-cancel: var(--color-greyBlue);
   --color-cancel--hover: var(--color-greyBlue--dark);
```

Comment:
Originally planned to remove cancel color in this PR, preserving to prevent a breaking change in mobile

Reply:
Speaking of... we have to update the mobile app now too, eh?

Reply:
Yep, but @agglasaur is working on some changes to Button so there's going to be work done in there anyways 😅

Reply:
I am cool with that answer